test/unit/services/cms_reset_service_test.rb

```diff
@@ -16,6 +16,7 @@ def test_call
     page = provider.pages.first!
 
     assert @cms_reset.call(provider)
+    provider.reload
```

Comment:
it could also be in the next line as `refute_equal page, provider.reload.pages.first!`, but either way it is fine :smile:

Reply:
Btw, this could also be an opportunity to ensure not having the postgres random order problem :smile:

Reply:
Yeah ... but this problem I think we should track it  in JIRA. Maybe there are some options for Postgres to have a default order